docs/source/apps/aces/aces_qss.ipynb

```diff
@@ -0,0 +1,319 @@
+{
```

Comment:
Equivalent comments for the `qss` notebook from the `css` notebook review comments

Reply:
Done.